### PR TITLE
Homebrew location for appium webdriver agent

### PR DESCRIPTION
**Pull request by vamvindev**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by jlipps — APPROVED** [review]

**Comment by jlipps** [comment]
since all this documentation is going away, probably want to propose the same change to the xcuitest repository

File: docs/en/advanced-concepts/wda-custom-server.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Also, it might be necessary to create an empty folder for WDA resources:
 mkdir -p /usr/local/lib/node_modules/appium/node_modules/appium-webdriveragent/Resources/WebDriverAgent.bundle
 ```
 
+If you've installed npm through brew install, then all the sub libraries installed through npm will be under homebrew directory. Hence, the Webdriver agent for appium will be located here 
+
+``` 
+/opt/homebrew/lib/node_modules/appium/node_modules/appium-webdriveragent
```
Comment (KazuCocoa) [inline docs/en/advanced-concepts/wda-custom-server.md:38]: This path is probably for macOS 12 (when M1 released), so perhaps it needs to address homebrew version, or macOS version that the path is the default one.

**Comment by boneskull** [comment]
Thanks for the PR!

My opinion:

homebrew tends to do big breaking changes often enough that I'd be wary of putting any homebrew-specific information in our docs other than a `brew install` command, for fear of it quickly becoming out-of-date.  Though I do agree it's useful, maybe a compromise would be to use its own facilities to retrieve base paths, e.g., `HOMEBREW_PREFIX` or `brew config`

**Comment by jlipps** [comment]
Yeah, we should probably phrase this in terms of brew config or some CLI tool or env var to determine the brew prefix

**Pull request closed without merging**